scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -425,23 +485,34 @@ Function RunQuickstartCertsTest
         -n `"$env:computername`" ``
         -r `"$ContainerRegistry`" ``
         -u `"$ContainerRegistryUsername`" ``
-        -p `"$ContainerRegistryPassword`" --optimize_for_performance true ``
+        -p `"$ContainerRegistryPassword`" ``
         -t `"${ArtifactImageBuildNumber}-windows-$(GetImageArchitectureLabel)`" ``
-        --leave-running=Core ``
+        --optimize_for_performance true ``
+        --leave-running=All ``
```

Comment:
why change from "--leaving-running=Core" to "--leaving-running=All"?

Reply:
Setting it to `Core` causes IotEdgeQuickstart to send out a new deployment based on its _internal, hard-coded deployment JSON_--it doesn't use the deployment file you specified. In the two cases where I set `All`, we're using `runtime_only_deployment.template.json`, which doesn't deploy any modules besides Edge Agent and Hub anyway, so no need to do another deployment. `All` simply disables that functionality.

Reply:
I am ok with it, but why not fix current implementation to take deployment file?

Reply:
I wrestled with that for a while...but I didn't want to fix the current impl in this particular case because the current impl is about to change. A lot. :)